include/highfive/bits/H5Slice_traits.hpp

```diff
@@ -27,12 +27,12 @@ class ElementSet {
     ///
     /// \param list List of continuous coordinates (e.g.: in 2 dimensions space
     /// `ElementSet{1, 2, 3 ,4}` creates points `(1, 2)` and `(3, 4)`).
-    explicit ElementSet(std::initializer_list<std::size_t> list);
+    ElementSet(std::initializer_list<std::size_t> list);
```

Comment:
Probably OK, because more permissive, meaning it shouldn't break anything and:
```
void foo(ElementSet);
foo({1, 2, 3});
```
seems fair.